### DISCUSSION TITLE
Navigation link broken

### DISCUSSION THREAD

**Discussion by ydv-ankit** [community/community#65674] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I was visiting this webpage URL https://docs.github.com/articles/about-github-s-use-of-your-data where it showed me 'NOT FOUND'.
Either this url should be updated or proper configs should be enabled for smooth user experience

**Comment by LvnReyS16** [comment]
I think the most likely explanation is that the URL is outdated. GitHub frequently updates their documentation, so it's possible that the article about their data use has been moved to a new URL.
